rust/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.java
```

Comment:
What's the purpose of adding *.java? These files are being generated automatically?

Reply:
Most java file are generated by `antlr *.g4`, base classes are not included by next line `!*Base.java`